.github/workflows/pr.yml

```diff
@@ -101,3 +101,24 @@ jobs:
       with:
         report_paths: unittests.xml
 
+  asana:
+    name: Create Asana Task
+    needs: [swiftlint, unit-tests]
```

Comment:
Why shellcheck is not included? Now I've noticed that it's also not included in macOS. We should probably include it too.

Reply:
@ayoy Good point, it was deliberated because I thought that we couldn't have flaky issues with shellcheck, but I guess the same could be said to I'll add it here and add to macOS later on. Thanks :)